.github/workflows/ci.yml

```diff
@@ -85,9 +93,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - id: unit-test-setup
```

Comment:
Switching Quarkus IT to integration-test approach so it can run tests against previously built artifacts